src/queries/circle/isMember.ts

```diff
@@ -0,0 +1,26 @@
+import { PRICE_STATE } from 'common/enums'
+import { CircleToIsMemberResolver } from 'definitions'
+
+const resolver: CircleToIsMemberResolver = async (
+  { id },
+  _,
+  { viewer, dataSources: { atomService }, knex }
+) => {
+  if (!viewer.id) {
+    return false
+  }
+
+  const records = await knex
+    .select()
+    .from('circle_subscription_item as csi')
```

Comment:
only items attached to the active subscription?

Reply:
Added.

Reply:
Added.